<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>398</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="receiverIP_Text">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Receiver's IP :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="receiverIP_Input">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="receiverName_Input">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="receiverName_Text">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Receiver's Name :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="registeredUserList">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseTextFileButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Erased Saved User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addReceiverButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>ADD RECEIVER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeSelectedButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Remove Selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="registeredUserText">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Registered Receivers</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clearScreenButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Clear Screen</string>
    </property>
   </widget>
   <widget class="QLabel" name="description_Text">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow up to 2000 characters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendingButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yourMessage_Input">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="maxLength">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QLabel" name="yourMessage_Text">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Your Message</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
